frontend/src/metabase/admin/app/selectors.ts

```diff
@@ -32,3 +32,6 @@ export const isDeprecationNoticeEnabled = (state: State): boolean => {
     state.settings?.values?.["deprecation-notice-version"]
   );
 };
+
+export const getCurrentVersion = (state: State) =>
+  state.settings?.values.version?.tag ?? "";
```

Comment:
this was the only selector left in settings, so I moved it up to this other selectors file